VillanOrHero/Controllers/QuestionViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ViewController.swift
+//  MarvelVillanOrHero
+//
+//  Created by Kurs on 27/08/2020.
+//  Copyright © 2020 Kurs. All rights reserved.
+//
+
+import UIKit
+
+class QuestionViewController: UIViewController {
+
+    @IBOutlet private weak var userName: UILabel!
+    @IBOutlet private weak var questionLabel: UILabel!
+    @IBOutlet private weak var image: UIImageView!
+    @IBOutlet private weak var scoreLabel: UILabel!
+    @IBOutlet private weak var questionNumberLabel: UILabel!
+    @IBOutlet private weak var backGroundLabel: UILabel!
+    
+    var heroAlignment = ""
+    var timer: Timer?
+    var timePoints = 5.00
+
+    var characterManager = CharacterManager()
+    var questionManager = QuestionManager.sharedInstance
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        characterManager.delegate = self
+        characterManager.fetchCharacter()
+        userName.text = questionManager.userName
+    }
+    
+    @IBAction func buttonPressed(_ sender: UIButton) {
+        if questionManager.questionNumber == 5 {
+            checkAnswerAndChangeUIColor(sender: sender)
+            questionNumberLabel.text = "\(questionManager.questionNumber)/5"
+            questionManager.addScoreToUserDefault(with: questionManager.userName, and: questionManager.userResult)
+            questionManager.stopTimer()
+            performSegue(withIdentifier: Constants.scoreVCSegue, sender: self)
+        } else {
+            checkAnswerAndChangeUIColor(sender: sender)
+            characterManager.fetchCharacter()
+            scoreLabel.text = "Score: \(questionManager.roundScore())"
+            questionNumberLabel.text = "\(questionManager.questionNumber)/5"
+            questionManager.stopTimer()
+
+        }
+    }
+    func checkAnswerAndChangeUIColor(sender: UIButton) {
+        if questionManager.checkAnswer(sender.accessibilityIdentifier!, characterAlignment: heroAlignment) {
+            backGroundLabel.backgroundColor = #colorLiteral(red: 0.1568627451, green: 0.8745098039, blue: 0.6, alpha: 1)
```

Comment:
We can create another struct or class to declare common colors so that we can reuse later, like your Constants struct.